integration-cli/docker_cli_version_test.go

```diff
@@ -21,9 +22,7 @@ func (s *DockerSuite) TestVersionEnsureSucceeds(c *check.C) {
 	}
 
 	for k, v := range stringsToCheck {
-		if strings.Count(out, k) != v {
-			c.Errorf("%v expected %d instances found %d", k, v, strings.Count(out, k))
-		}
+		c.Assert(strings.Count(out, k), checker.Equals, v, check.Commentf("%v expected %d instances found %d", k, v, strings.Count(out, k)))
```

Comment:
@MHBauer Thanks, how about simply using "check.commentf("%v in %s do not match excepted", k, out)"？